src/ocm/components/clusterConfiguration/HostInventory.tsx

```diff
@@ -30,32 +32,43 @@ import { useClusterSupportedPlatforms } from '../../hooks';
 import { OcmSwitchField } from '../ui/OcmFormFields';
 import { selectCurrentClusterPermissionsState } from '../../selectors';
 
-const PlatformIntegrationLabel: React.FC = () => (
-  <>
-    <span>Integrate with vSphere</span>{' '}
-    <PopoverIcon
-      bodyContent={
-        <>
-          Enable vSphere integration to access features like node auto-scaling and persistent
-          storage directly inside OpenShift. You'll need to set vSphere configuration after cluster
-          installation is complete.
-        </>
-      }
-      footerContent={
-        <>
-          <Title headingLevel="h6">Requirements</Title>
-          <List>
-            <ListItem>A network connection between vSphere and the installed OCP cluster</ListItem>
-            <ListItem>
-              Set <code>disk.enableUUID</code> to <code>true</code> inside vSphere
-            </ListItem>
-          </List>
-        </>
-      }
-      buttonOuiaId="platform-integration-vSphere-popover"
-    />
-  </>
-);
+const PlatformIntegrationLabel: React.FC<{
+  isPlatformTypeVsphere: boolean;
```

Comment:
We shouldn't be checking `platformType` here, as these messages appear when the integration is enabled, but not active.
`platformType=X` represents the integration being active (and obviously enabled)